packages/route-viewer-overlay/src/index.tsx

```diff
@@ -117,16 +117,17 @@ const RouteViewerOverlay = (props: Props): JSX.Element => {
 
     patterns.forEach(ptn => {
       ptn.stops?.forEach(stop => {
-        if (stop.geometries?.geoJson?.type === "Polygon") {
```

Comment:
Could this not have been resolved by just adding a question mark here? I guess it's cleaner now...